### DISCUSSION TITLE
Im getting this error: “error:”requested path is invalid” when using my URL

### DISCUSSION THREAD

**Discussion by JessAle07** [supabase/supabase#30394] [Questions] [answered]
Hey guys, so, as the title says, im currently having this issue when i try to use my supabase URL, getting that error i pu on the title. Its my first time using this database (or any database) and im lost. Im from colombia so i set up my server to be on brazil, created a new table and nothing else. I tried using the URL on a vercel project and then i found the URL from supabase its the problem. So here i am, really lost and hoping for some help. Thanks everyone!

**Comment by GaryAustin1** [comment]
I responded to this on Discord yesterday.  You will need to provide more info for anyone to help you.  What call are you making?  What URL are you using where?

**Reply by JessAle07** [reply, marked as answer]
Oh yeah, sorry for that, I was kind of desperate, but I managed to fix it myself. Basically I was trying to access the URL through the google bar (yes, dumb, but I had never done that before) and well, after some changes and environment variables on vercel, I could fix it. Thanks anyway, really appreciate it comment 🙏

**Comment by phillipshaong** [comment]
Hi, I am also getting a similar error. I followed the instructions from https://supabase.com/docs/guides/auth/social-login/auth-google?queryGroups=platform&platform=flutter. I am deploying a flutter app. I tripled check everything, but it is giving me this error. 

![CleanShot 2025-02-14 at 02 36 00](https://github.com/user-attachments/assets/6c3de323-99d5-4700-91e2-b6630100fb72)

Below is my configuration: 

![CleanShot 2025-02-14 at 02 34 00](https://github.com/user-attachments/assets/481bfc16-c2fe-4955-aa52-4e2d5b4417e5)
![CleanShot 2025-02-14 at 02 35 17](https://github.com/user-attachments/assets/77c64c86-ae05-49e8-8dc6-65552f77b955)

Code:

![CleanShot 2025-02-14 at 02 36 26](https://github.com/user-attachments/assets/ce27a31a-1c29-4769-b6dd-f5b26ee971a6)


https://github.com/user-attachments/assets/6983549f-23d3-43c0-99a0-842d93375409

**Reply by chiragksharma** [reply]
it is redirecting to the wrong uri 
Can you remove that firebase uri1 from the `Authorized redirect URIs` ?

**Reply by phillipshaong** [reply]
Done, it is still giving the same error. 

![CleanShot 2025-02-14 at 13 03 55](https://github.com/user-attachments/assets/25c83243-da20-433f-ac10-3e629dee24ad)

**Comment by phillipshaong** [comment]
There is an obscure setting that is NOT mentioned in the configuration of OAuth. You must configure your Redirect URL's on the **supabase** side (https://supabase.com/docs/guides/auth/redirect-urls), to allow which redirects are allowed from the code. There needs to be documentation, especially in the code examples, where the parameter or `redirectTo` is listed in the dashboard https://supabase.com/dashboard/project/<project_id>/auth/url-configuration

![CleanShot 2025-02-14 at 13 53 06](https://github.com/user-attachments/assets/3137f109-41a8-4e07-89f8-14524b08e5a9)

**Reply by GaryAustin1** [reply]
![image](https://github.com/user-attachments/assets/bdaac5fa-37fe-4946-9c13-44db4eed6bc0)
![image](https://github.com/user-attachments/assets/a59f2628-f148-4317-a341-ba5ca60bcfae)
![image](https://github.com/user-attachments/assets/ff7fb3b4-adf0-47ac-803f-502963d0f851)

It is documented in many places (although I think a recent change has the wrong link in the first two).

But if you have feedback or have a specific place you think it should be besides those you should generate an issue in supabase/supabase issues or a PR to add it to the docs.   Supabase will not likely see your comment here.

**Reply by phillipshaong** [reply]
I see it now. Though, the error message was not very helpful. It would be nice to get more detailed message, like "RedirectUrl is not allowed via configuration."
